index.php

```diff
@@ -2,31 +2,34 @@
 <html lang="en">
 <?php
 
+date_default_timezone_set("America/New_York");
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+$app = new Silex\Application();
+
+$app['debug']=true;
+
 use ABS\Classes\ModelClass as ModelClass;
 use ABS\Classes\ControllerClass as ControllerClass;
-use ABS\Classes\FooterViewClass as FooterViewClass;
-use ABS\Classes\HeaderViewClass as HeaderViewClass;
-use ABS\Classes\HeadViewClass as HeadViewClass;
-use ABS\Classes\MainViewClass as MainViewClass;
-
-require_once __DIR__.'/app/start.php';
 
 $model = new ModelClass();
-$headView = new HeadViewClass($model);
-$headerView = new HeaderViewClass($model);
-$mainView= new MainViewClass($model);
-$footerView = new FooterViewClass($model);
 $controller= new ControllerClass($model);
 
 if (isset($_GET['action']))
 {
 	$controller->{$_GET['action']}();
 }
 
-echo $headView->output();
-echo $headerView->output();
-echo $mainView->output($model->page);
-echo $footerView->output();
+$app->register(new Silex\Provider\TwigServiceProvider(), array( 
+		'twig.path' => __DIR__.'/views',
+	));
+
+$app->get('/', function () use ($app, $model) {
+	return $app['twig']->render($model->page, []);
+});
+
+$app->run();
 
 ?>
```

Comment:
remove

"If a file is pure PHP code, it is preferable to omit the PHP closing tag at the end of the file. This prevents accidental whitespace or new lines being added after the PHP closing tag, which may cause unwanted effects because PHP will start output buffering when there is no intention from the programmer to send any output at that point in the script."

from: http://php.net/manual/en/language.basic-syntax.phptags.php

Reply:
done